Menu.md

```diff
@@ -0,0 +1,413 @@
+        "label": "Clothing",
+        "link": "/Clothing/Mens?map=c%2CspecificationFilter_7721",
+        "child": "39216"
+      },
+      {
+        "id": "41202",
+        "label": "Footwear",
+        "link": "/footwear/Mens?map=c%2CspecificationFilter_7721",
+        // (...)
+      },
+      {
+        "id": "41203",
+        "label": "Accessories",
+        "link": "/clothing/accessories/Mens?map=c%2Cc%2CspecificationFilter_7721",
+        // (...)
+      }
+    ]
+  },
+  
+  "mens-clothing": {
+    "id": "39216",
+    "title": "Men's Clothing",
+    "items": [
+      {
+        "id": "41201",
+        "label": "Jackets",
+        "link": "/Clothing/Jackets/Mens?map=c%2Cc%2CspecificationFilter_7721",
+      },
+      {
+        "id": "41202",
+        "label": "Shirts",
+        "link": "/Clothing/Shirts/Mens?map=c%2Cc%2CspecificationFilter_7721",
+      },
+      {
+        "id": "41203",
+        "label": "Hoodies & Sweaters",
+        "link": "/Clothing/Hoodies---Sweatshirts/Mens?map=c%2Cc%2CspecificationFilter_7721",
+      },
+      {
+        "id": "41204",
+        "label": "Pants",
+        "link": "/clothing/Mens/pants?map=c,specificationFilter_7721,c"
+      },
+      // (...)
+    ]
+  }
+}
+```
+
+Then simply use the `menu`block in their theme, but now in a much simpler way, since a lot of the customisation of individual items will happen via CMS:
+
+```jsonc
+{
+  "menu": {
+    "props": {
+      "mainNavigation": {
+        // Notice this id comes from the navigation.json file
+        // and matches a certain navigation.
+        id: 39214,
+        // This menu-item.root block will be used to render
+        // the items from the navigation referenced above.
+        Item: 'menu-item.root'
+      }
+    },
+  },
+  "menu-item.root": {
+    // (...)
+  }
+}
+```
+
+With the two bits of code above, you should be able to generate a simple navigation, which represents the first level of the menu we're trying to achieve:
+
+![als-menu-main-navigation](https://user-images.githubusercontent.com/27777263/77101988-ee5a7b80-69f6-11ea-87fb-412fb4cc4797.png)
+
+From this point on, the customisation of each submenu inside each of the `menu-item`s is done using the CMS and implemented `submenu`blocks. For the example we're working on, let's consider the submenu shown in the first image of the menu, where we have a **Featured** list of items and a few other lists which look all the same. To represent this kind of layout, a `submenu`block should be implemented, such as a `submenu.default`:
+
+```jsonc
+{
+  // All of this configuration should be made via CMS
+  "submenu.default": {
+    "mainNavigation": {
+      // Notice that there is no id attribute
+      // for this navigation. This will be explained
+      // in the Technical Details section.
+      "Item": "menu-item.list"
+    },
+    "featuredNavigation": {
+      "id": 1234,
+      "Item": "menu-item.featured"
+    },
+    "bottomText": "Free shipping on orders over $45"
```

Comment:
This should be inside of `"props"` right?

Reply:
Yes! Good catch